source/16-7-2_Adding_and_Editing_Profile_Custom_Tabs_and_Fields.markdown

```diff
@@ -6,136 +6,181 @@
+
+![Editing Fields](16.7.2f.png)
+
+- Their names will already be entered into the form, but you will need to add the other details for them, such as type of field, as described for a single tab above.
+- When you've finished editing each one, click 'Update'. (Again, you still need to click on 'save all changes' before exiting the page.) When you have added all the details you can add options to the fields that need them. 
+
+### Setting Options
    
-In the right column, you’ll now see the settings for your field (if you can't, click on the field name - the grey button in the middle column.)
+The field that you have just added may require some options (for example, if you've added a 'Current status' field, you might want the options 'Active' and Inactive'.) If it's a field that you have just added it you will see buttons allowing you to add options. If it is an existing field that you are editing, click on the field name then select the three dots ('more' button) on the right to see these 'add' options. 
+
+![Adding Options](16.7.2g.png)
+
+#### Add a Single Option
 
-- This text in the right column will show you the name, and type of field, and if access to it is restricted.  If it’s a field that requires no options, such as ‘Date of Birth’ then that’s all you’ll see.  
-- If you can enter options you’ll see a ‘click to add’ button, and a text box.
-- The easiest way to enter your options is using the text box at the bottom of the column. Type or paste your options in here, separated by semi-colons (no spaces), then click the 'Add' button below the box to save them all. 
+- To add your options one at a time, click on the 'add new option' button. This opens a pop-up box where you add the text of your option (in the previous example this would be 'Active'). 
+- Click 'Update' when you have finished. 
+- Continue adding all the options you need in this way.
 
-![Adding Options to a Field](16.7.2h.png)
+#### Add a Number of Options at Once
 
-It is also possible to add them one at a time by using the ‘click to add’ button. 
+- If you have a field which provides a number of different options, click on 'bulk add option'. 
+- Type all your options in the text box, with a semi-colon (;) between each one. You do not need a space before each new one. For example, 'Active;Inactive;Not known.'
+- When you have finished, click 'add all'. They will appear in a list below the field name.
 
-You'll see your options appear in the list under 'Options'. 
+#### Add from a Built-in List
 
-![Final Option Text in Field](16.7.2i.png)
+- Lamplight contains a number of built-in sets of options. To see what is there, click on the 'add from built-in list' button. They include yes/no; some demographic options such as ethnicity, gender, religion and language; number lists; days and months. If you would like to use any of these lists as options for your field, click on the circle next to the appropriate one then click 'add'. 
```

Comment:
You can edit or add to these items once they have been added to a field, if needed.